com.unity.netcode.adapter.utp/Tests/Runtime/ConnectionTests.cs

```diff
@@ -17,6 +17,23 @@ public class ConnectionTests
         private List<TransportEvent> m_ServerEvents;
         private List<TransportEvent>[] m_ClientsEvents = new List<TransportEvent>[k_NumClients];
 
+        private IEnumerator WaitForAllClientsConnected()
+        {
+            for (int i = 0; i < k_NumClients; i++)
+            {
+                if (m_ClientsEvents[i].Count == 0)
+                {
+                    yield return WaitForNetworkEvent(NetworkEvent.Connect, m_ClientsEvents[i]);
+                }
+            }
+
+            // Check that all clients received the correct event.
+            Assert.True(m_ClientsEvents.All(evs => evs[0].Type == NetworkEvent.Connect));
```

Comment:
nit: Just a tip, you can use Assert.That and specify constraints. This will ensure that you get a relevant error message when the test fails, rather than just "Expected true, got false".

https://docs.nunit.org/articles/nunit/writing-tests/assertions/assertion-models/constraint.html

Reply:
I didn't know about that. Thanks for pointing it out!